src/mame/drivers/sys1121.cpp

```diff
@@ -0,0 +1,64 @@
+// license:BSD-3-Clause
+// copyright-holders:Katherine Rohl
+/*
+ *  Motorola SYS1121 VME chassis. 
+ *
+ *  The basic configuration was an MVME12x MPU,
+ *  an MVME050 system controller, and an MVME320 disk
+ *  controller.
+ */
+ 
+#include "emu.h"
+#include "bus/vme/vme_mvme120.h"
+#include "logmacro.h"
+
+namespace {
+
+class sys1121_state : public driver_device
+{
+	public:
+	sys1121_state(const machine_config &mconfig, device_type type, const char *tag)
+			: driver_device (mconfig, type, tag)
+		{
+		}
+
+		void sys1121(machine_config &config);
+		
+	protected:
+		virtual void machine_start() override { }
+	//  virtual void machine_reset() override;
```

Comment:
The indentation here is all over the place.  Please outdent the accessibility levels and use one tab per scope level.  Also, you don’t need to override `machine_start` if you aren’t doing anything.